src/sdk/objections/service.ts

```diff
@@ -0,0 +1,82 @@
+import { AxiosError, AxiosRequestConfig, AxiosResponse, Method } from "axios";
+import axios from "axios";
+import logger from "../../utils/logger";
+import { INTERNAL_API_URL } from "../../utils/properties";
+import { ApiError, ObjectionPatch } from "./types";
+
+const HTTP_POST: Method = "post";
+
+/**
+ * An axios config with common elements for API calls.
+ * @param {Method} httpMethod the http method to use eg. get, post
+ * @param {string} url of the api endpoint
+ * @param {string} token Bearer token for API call
+ */
+const getAxiosRequestConfig = (httpMethod: Method, url: string, bearerToken: string): AxiosRequestConfig => {
```

Comment:
Could the axios calls go into a separate file?

Reply:
yes I can look at that

Reply:
extracted axios out into axios.client.ts